.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: julia
+os:
+  - linux
+  - osx
+  - windows
+julia:
+  - 1.3
+  - nightly
+notifications:
+  email: false
+after_success:
+  - julia -e 'using Pkg; Pkg.add("Coverage"); using Coverage; Coveralls.submit(Coveralls.process_folder())';
```

Comment:
Can simply be a `coveralls: true` at the top level as well https://docs.travis-ci.com/user/languages/julia/#coverage